File: Contest/2contest/Mini-MaxSum.cpp
#include <iostream>

using namespace std;

int main(){
    string s;
    int a[5];
    getline (cin, s);
    s = s + " ";
    int p;
    for(int i = 0; i < 5; i++){
        p = s.find(" ");
        a[i] = stol(s.substr(0, p));
        s = s.substr(p+1, s.length());
    }
    int sum = 0;
    int max;
    int min;
    for(int i = 0; i < 5; i++){
        sum = 0;
        for(int j = 0; j < 5; j++){
            if(i == j) continue;
            sum += a[j];
        }
        if(i == 0){
            max = sum;
            min = sum;
        }
        if(sum > max) max = sum;
        if(sum < min) min = sum;
    }
    cout << min << " " << max;
}

File: Contest/2contest/Mini-MaxSum.py
s = input() + " "
a = [0] * 5
for i in range(5):
    p = s.find(" ")
    a[i] = int(s[0:p])
    s = s[p+1:len(s)]
max
min
for i in range(5):
    sum = 0
    for j in range(5):
        if(i == j):
            continue
        sum += a[j]
    if i == 0:
        max = sum
        min = sum
    if sum > max:
        max = sum
    if sum < min:
        min = sum
s= str(min) +  " " + str(max)
print(s)